actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -26,7 +26,100 @@ module TagHelper
       PRE_CONTENT_STRINGS[:textarea]  = "\n"
       PRE_CONTENT_STRINGS["textarea"] = "\n"
 
+      class TagBuilder #:nodoc:
+        include TagHelper
 
+        VOID_ELEMENTS = %i(base br col embed hr img input keygen link meta
+        param source track wbr).to_set
+
+        def initialize(view_context)
+          @view_context = view_context
+        end
+
+        private
+          def tag_string(name, content = nil, escape_attributes: true, **options, &block)
+            content = @view_context.capture(self, &block) if block_given?
+            if VOID_ELEMENTS.include?(name) && content.nil?
+              "<#{name}#{tag_options(options, escape_attributes)}>".html_safe
+            else
+              content_tag_string(name, content || '', options, escape_attributes)
+            end
+          end
+
+          def respond_to_missing?(method_name, include_private = false)
```

Comment:
Since we don't care about the args we can replace them with: `respond_to_missing?(*)`.

Is there a `respond_to?` test?